src/app/Library/CrudPanel/Traits/Create.php

```diff
@@ -225,29 +173,41 @@ private function createUpdateOrDeleteOneToOneRelation($relation, $relationMethod
      */
     private function attachManyRelation($item, $relation, $relationDetails, $relation_values)
     {
-        $model_instance = $relation->getRelated();
-        $relation_foreign_key = $relation->getForeignKeyName();
-        $relation_local_key = $relation->getLocalKeyName();
+        $modelInstance = $relation->getRelated();
+        $relationForeignKey = $relation->getForeignKeyName();
+        $relationLocalKey = $relation->getLocalKeyName();
 
         if ($relation_values === null) {
             // the developer cleared the selection
             // we gonna clear all related values by setting up the value to the fallback id, to null or delete.
-            $removed_entries = $model_instance->where($relation_foreign_key, $item->{$relation_local_key});
+            $removed_entries = $modelInstance->where($relationForeignKey, $item->{$relationLocalKey});
 
-            return $this->handleManyRelationItemRemoval($model_instance, $removed_entries, $relationDetails, $relation_foreign_key);
+            return $this->handleManyRelationItemRemoval($modelInstance, $removed_entries, $relationDetails, $relationForeignKey);
         }
-        // we add the new values into the relation
-        $model_instance->whereIn($model_instance->getKeyName(), $relation_values)
-            ->update([$relation_foreign_key => $item->{$relation_local_key}]);
+        // we add the new values into the relation, if it is HasMany we only update the foreign_key,
```

Comment:
This segment adds support for MorphMany selectable. I was only accounting for HasMany.